internal/config/config.go

```diff
@@ -147,9 +147,14 @@ type Defaults struct {
 	Timeout time.Duration `yaml:"timeout,omitempty" json:"timeout"`
 }
 
-// Instrumentation represents instrumentation options
+// OverwriteDeviceSettings represents override settings.
+type OverwriteDeviceSettings struct {
```

Comment:
That's a bit of a mouthful 😅 

These settings are actually referred to as application settings:
![Screen Shot 2022-04-18 at 3 41 46 PM](https://user-images.githubusercontent.com/1869292/163889113-7824066c-378e-41b7-b62c-502ffe9664b3.png)

So maybe 
```suggestion
type AppSettings struct {
```

is more apt?

Reply:
looks better!